scripts/remove

```diff
@@ -1,31 +1,35 @@
 #!/bin/bash
 
+# Source local helpers
+source ./_common.sh
+
+# Source app helpers
+source /usr/share/yunohost/helpers
+
 # Set app specific variables
 app="$YNH_APP_INSTANCE_NAME"
 dbname=$app
 dbuser=$app
 
-# Source YunoHost helpers
-source /usr/share/yunohost/helpers
-
 # Drop MySQL database and user
-ynh_mysql_drop_db "$dbname" || true
-ynh_mysql_drop_user "$dbuser" || true
+ynh_mysql_drop_db "$dbname"
+ynh_mysql_drop_user "$dbuser"
 
 # Retrieve domain from app settings
 domain=$(ynh_app_setting_get "$app" domain)
 
 # Delete app directory and configurations
-sudo rm -rf "/var/www/${app}" "/var/log/${app}"
-sudo rm -f "/etc/php5/fpm/pool.d/${app}.conf"
-[[ -n $domain ]] && sudo rm -f "/etc/nginx/conf.d/${domain}.d/${app}.conf"
+ynh_secure_remove "/var/www/${app}"
+ynh_secure_remove "/var/log/${app}"
+ynh_remove_fpm_config
+ynh_remove_nginx_config
+
+# Remove metapackage and its dependencies
+ynh_remove_app_dependencies
 
 # Reload services
-sudo service php5-fpm restart || true
-sudo service nginx reload || true
+service php5-fpm restart
+service nginx reload
 
 # Remove the user account
-# (must be done after php-fpm restart)
-id "$app" >/dev/null 2>&1 \
-  && sudo deluser --quiet --remove-home "$app" >/dev/null \
-  || true
+ynh_system_user_delete "$app"
```

Comment:
This whole script need to be reworked to according to https://github.com/YunoHost/example_ynh/blob/master/scripts/remove